peering/tables.py

```diff
@@ -122,7 +122,7 @@ class BGPSessionStateColumn(tables.TemplateColumn):
     def __init__(self, *args, **kwargs):
         default = kwargs.pop("default", "")
         visible = kwargs.pop(
-            "visible", settings.NAPALM_USERNAME and settings.NAPALM_PASSWORD
+            "visible", Router.napalm_username and Router.napalm_password
```

Comment:
This won't work. We need to perform this check against a router instance.

Reply:
I have changed it with a if, but i'm quite not sure if that will work.